doc/usage/getting_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting started
+
+Currently we don't publish the binary build with the release, but it is pretty straight forward to build it by following the steps mentioned [here](../development/local_setup.md#build). But we do publish the docker image with each release, please check the [release page](https://github.com/gardener/etcd-backup-restore/releases) for the same.
+
+## Usage
+
+You can follow the `help` flag on `etcdbrctl` command and its sub-commands to know the usage details. Some of the common use cases are mentioned below. Although examples below uses AWS S3 as storage provider, we have added support for AWS, GCS, Azure, Openstack swift and Alicloud OSS object store. It also supports local disk as storage provider.
+
+### Cloud Provider Credentials
+
+The procedure to provide credentials to access the cloud provider object store varies for different providers.
+
+* For `AWS S3`, the `credentials` file has to be provided in the `~/.aws` directory.
+
+* For `GCP Containers`, the service account json file should be provided in the `~/.gcp` as a `service-account-file.json` file.
+
+* For `Azure Blob storage`, `STORAGE_ACCOUNT` and `STORAGE_KEY` should be made available as environment variables.
+
+* For `Openstack Swift`, `OS_USERNAME`, `OS_PASSWORD`, `OS_AUTH_URL`, `OS_TENANT_ID` and `OS_DOMAIN_ID` should be made available as environment variables.
+
+* For `Alicloud OSS`, `ALICLOUD_ENDPOINT`, `ALICLOUD_ACCESS_KEY_ID`, `ALICLOUD_ACCESS_KEY_SECRET` should be made available as environment variables.
+
+### Taking scheduled snapshot
+
+`etcd` should already be running. One can apply standard cron format scheduling for regular backup of etcd. The cron schedule is used to take full backups. The delta snapshots are taken at regular intervals in the period in between full snapshots as indicated by the `delta-snapshot-period-seconds` flag. The default for the same is 10 seconds.
+
+etcd-backup-restore has two garbage collection policies to collect existing backups from the cloud bucket. The flag `garbage-collection-policy` is used to indicate the correct garbage collection policy.
+1. `Exponential`
+1. `LimitBased`
+
+If using `LimitBased` policy, the `max-backups` flag should be provided to indicate the number of recent backups to persist at each garbage collection cycle.
+
+```console
+$ ./bin/etcdbrctl snapshot --storage-provider="S3" --etcd-endpoints http://localhost:2379 --schedule "*/1 * * * *" --store-container="etcd-backup" --delta-snapshot-period-seconds=10 --max-backups=10 --garbage-collection-policy='LimitBased'
+INFO[0000] Validating schedule...
```

Comment:
```suggestion
INFO[0000] etcd-backup-restore Version: x.y.z-dev
INFO[0000] Git SHA: abcdef12
INFO[0000] Go Version: go1.12.x
INFO[0000] Go OS/Arch: darwin/amd64
INFO[0000] Validating schedule...
```

Reply:
I purposefully didn't address this, cause output might change with version. So, its better to hardcode the output here.